### PR TITLE
[LTD-1342] Move case forward

### PR DESCRIPTION
**Pull request by alixedi**
We were using system flags to ensure that the case is only moved to countersigning queue when we have advice for all destinations.

While working on it, I realized - the routing is run when the user presses the `I am Done` button. What if we could hide/disable the button until we have advice for all the destinations?

If we could do this (this PR), we don’t need any routing changes or create new system flags, etc.

Just one little thing that is bothering me - when this button is disabled, we should probably display a message that tells users why it is disabled.

### CODE REVIEW

File: caseworker/advice/templates/advice/view_my_advice.html [hunk 1]
```diff
@@ -5,6 +5,20 @@
 <div class="govuk-width-container">
     <a href="#" class="govuk-back-link">Back</a>
     <main class="govuk-main-wrapper">
+        {% for error in form.non_field_errors %}
```
Comment (saruniitr) [inline caseworker/advice/templates/advice/view_my_advice.html:8]: is it done like this because it is just a template and not a form?

Comment (alixedi) [reply caseworker/advice/templates/advice/view_my_advice.html:8]: It is a form but I wanted to show the errors at a different place.

File: unit_tests/caseworker/advice/views/test_view_my_advice_view.py [hunk 2]
```diff
@@ -19,6 +19,35 @@ def url(data_queue, data_standard_case):
...
+            "level": "user",
+            "note": "additional notes",
+            "proviso": "no conditions",
+            "text": "meets the criteria",
+            "third_party": "95c2d6b7-5cfd-47e8-b3c8-dc76e1ac9747",
+            "type": {"key": "proviso", "value": "Proviso"},
+            "ultimate_end_user": None,
+            "user": current_user,
+        }
+        for good_id in ("0bedd1c3-cf97-4aad-b711-d5c9a9f4586e", "6daad1c3-cf97-4aad-b711-d5c9a9f4586e")
```
Comment (wkeeling) [inline unit_tests/caseworker/advice/views/test_view_my_advice_view.py:47]: Doesn't look as though `good_id` is being used in the fixture data, but that probably doesn't matter for the test?

Comment (saruniitr) [inline unit_tests/caseworker/advice/views/test_view_my_advice_view.py:47]: `good_id` is not used

Comment (alixedi) [reply unit_tests/caseworker/advice/views/test_view_my_advice_view.py:47]: Good catch. It doesn't matter but it's probably best to use it.

Comment (alixedi) [reply unit_tests/caseworker/advice/views/test_view_my_advice_view.py:47]: Good catch. Using it now.

**Review by saruniitr — COMMENTED** [review]
Could you add a test to check "Move case forward" is not rendered on the template if advice is not completed? similarly that it is visible when advice is completed.
We can extend it later when advice is sent back etc

**Review by saruniitr — APPROVED** [review]

**Pull request merged**